visualization/app/codeCharta/util/loadFilesValidationToErrorDialog.ts

```diff
@@ -0,0 +1,42 @@
+import { CCFileValidationResult } from "./fileValidator"
+
+export function loadFilesValidationToErrorDialog(fileValidationResults: CCFileValidationResult[]) {
+	const htmlMessages = []
+
+	const filesWithErrors = fileValidationResults.filter(validationResult => validationResult.errors.length > 0)
+	if (filesWithErrors.length > 0) {
+		htmlMessages.push("<h2>Errors</h2>")
+		for (const fileWithErrors of filesWithErrors) {
+			const fileErrorMessage = buildFileErrorMessage(fileWithErrors)
+			htmlMessages.push(fileErrorMessage)
+		}
+	}
+
+	const filesWithWarnings = fileValidationResults.filter(validationResult => validationResult.warnings.length > 0)
+	if (filesWithWarnings.length > 0) {
+		htmlMessages.push("<h2>Warnings</h2>")
+		for (const fileWithWarnings of filesWithWarnings) {
+			const fileWarningMessage = buildFileWarningMessage(fileWithWarnings)
+			htmlMessages.push(fileWarningMessage)
+		}
+	}
```

Comment:
NIT: It seems that we could extract the two blocks to separate methods as a short refactoring.

Reply:
done